keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -50,59 +50,62 @@ import java.util.function.BiFunction
  */
 class MemoryCloudDriverCache(
   private val cloudDriver: CloudDriverService,
-  private val meterRegistry: MeterRegistry
+  private val meterRegistry: MeterRegistry,
+  cacheProperties: CacheProperties = CacheProperties()
 ) : CloudDriverCache {
 
   private val securityGroupSummariesByIdOrName = Caffeine.newBuilder()
     .executor(IO.asExecutor())
-    .maximumSize(1000)
-    .expireAfterWrite(1, MINUTES)
+    .maximumSize(cacheProperties.caches["securityGroups"]?.maximumSize ?: 1000)
```

Comment:
At some point, I think we should introduce a helper method to DRY up the instantiation of these caches.

Reply:
Done!